IrisSoftware/main.py

```diff
@@ -22,9 +22,11 @@ def __init__(self) -> None:
             self.shouldExit = False
             self.isCalibrated = False
             self.calibrationEyeCoords: list[list[tuple]] = []
+            self.faceBoxes = []
+            self.faceBox = None
             self.lastCursorPos = pyautogui.position()
             self.skipMouseMovement = False
-            self.interpolatorType = InterpolationType.LINEAR_REGRESSION #InterpolationType.JOYSTICK
+            self.interpolatorType = InterpolationType.TANGENT_LINEAR_REGRESSION # InterpolationType.TANGENT_LINEAR_REGRESSION
```

Comment:
```suggestion
            self.interpolatorType = InterpolationType.TANGENT_LINEAR_REGRESSION # InterpolationType.TANGENT_LINEAR_REGRESSION
```
Do we still need this comment here, or can we get rid of it?

Reply:
We can get rid of it